test/config/pg_types.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const should = require('should');
+const pgTypes = require('pg').types;
+const customTypes = require('../../config/pg_types');
+
+describe('parseDate', () => {
+  const DATE_OID = 1082;
+  const parseDate = pgTypes.getTypeParser(DATE_OID);
+
+  it('does not offset time', () => {
+    let testDate = '2016-12-01';
+    let expectedDate = new Date(testDate).toISOString();
+    let resultedDate = parseDate(testDate);
+    resultedDate.toISOString().should.equal(expectedDate);
+  });
+
+  it('returns null when value is null', () => {
+    let testDate = null;
+    should(parseDate(testDate)).equal(null);
+  });
+
+});
+
+describe('parseDateArray', () => {
```

Comment:
`it('calls parseDate() on single array items')`

For when you pass it an array of a single item. So we'd avoid creating an interval (unless we really want to).